core/src/main/java/io/questdb/cutlass/text/AbstractTextLexer.java

```diff
@@ -121,15 +123,20 @@ public void parseExactLines(long lo, long hi) {
                     if (zeroBytesWord == 0) {
                         ptr += 7;
                         this.fieldHi += 7;
+                        this.ascii = this.ascii && (word & NON_ASCII_MASK_FULL) == 0;
```

Comment:
This can be branchless:
```suggestion
                        this.ascii &= (word & NON_ASCII_MASK_FULL) == 0;
```

Reply:
this is done